src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -592,6 +622,41 @@ bool Terminal::SendCharEvent(const wchar_t ch, const WORD scanCode, const Contro
     return handledDown || handledUp;
 }
 
+void Microsoft::Terminal::Core::Terminal::_InvalidatePatternTree(interval_tree::IntervalTree<size_t, size_t>& tree)
+{
+    typedef interval_tree::IntervalTree<size_t, size_t> ThisTree;

```

Comment:
We could probably even move this typedef higher in the file, because we're probably not using any other variant of it other that `<size_t, size_t>`, right? That way we could re-use the one typedef throughout